quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/JavaOptsScriptTest.java

```diff
@@ -120,4 +120,14 @@ void testPicocliClosuresDisabled(LaunchResult result) {
         assertThat(output, containsString("DefaultFactory: groovy Closures in annotations are disabled and will not be loaded"));
     }
 
+    @Test
+    @Launch({"start-dev", "--optimized"})
```

Comment:
jfyi - AFAIK, there might be used `@EnabledOnOs` annotation.

Reply:
Updated to use that annotation and dryrun on all of the javaopts tests to save a couple of seconds.